packages/core/lib/database/saveErrorListRecord.ts

```diff
@@ -8,15 +11,20 @@ import insertErrorListRecord from "./insertErrorListRecord"
 import insertErrorListTriggers from "./insertErrorListTriggers"
 import updateErrorListRecord from "./updateErrorListRecord"
 import updateErrorListTriggers from "./updateErrorListTriggers"
-import type PhaseResult from "../../types/PhaseResult"
-import { getAnnotatedHearingOutcome } from "../../types/PhaseResult"
 
 const handleUpdate = async (db: Sql, recordId: number, result: PhaseResult): Promise<void> => {
   const aho = getAnnotatedHearingOutcome(result)
-  if (aho.Exceptions.length > 0) {
+  // If case is resubmitted to Phase 1 and case is ignored in Phase 2,
+  // we should update the record to set the correct error_status
+  if (aho.Exceptions.length > 0 || result.resultType === Phase2ResultType.ignored) {
```

Comment:
>If case is resubmitted to Phase 1

Is it the `aho.Exceptions.length > 0` that suggests it's resubmitted to Phase 1? If this is used for Phase 1 and 2 then it's not just if it's resubmitted to Phase 1 but also if Phase 2 generates an exception?

Reply:
No, it's not the exceptions length that indicates if the case is resubmitted. We don't check explicitly if the case is resubmitted here. This is to ensure that we don't end up having a case in Submitted status.

I haven't checked the Phase 1 code, but even if the case is ignored in Phase 1 after resubmission to Phase 1, then this code should cover it as well.